src/components/SelectWithLogo/SelectWithLogo.tsx

```diff
@@ -150,7 +150,7 @@ const OptionTemplateWithDataQa = (props: OptionTemplateWithDataQaProps) => {
       {...innerProps}
       ref={innerRef}
       data-qa-id={dataQa}
-      data-qa-props={dataQaProps}
+      data-qa-props={dataQaProps ? dataQaProps(props) : `value:${props.data.value}`}
```

Comment:
why do we update data-qa-props logic?

Reply:
the `data-qa-props` in the original component was `value:${props.data.value}` and when I check the output, it's iterating all the options value, so I return it to the original logic and add the props for customization